independent-publisher-connectors/Edatalia sign online/apiProperties.json

```diff
@@ -0,0 +1,43 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "endpoint":{
+        "type": "string",
+        "uiDefinition":{
+          "displayName": "Endpoint host",
+          "description": "Sandbox (restapi.firmar.info) Production (restapi.firmar.online)",
+          "tooltip": "Provide the end point host",
+          "constraints": {
+            "required": "true"
+          }
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0058a0",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "EndpointUrl",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('endpoint')"
+        }
+      }
+    ],
+    "publisher": "AXAZURE",
```

Comment:
@marpedroche The publisher value should be your name since your publishing as a independent publisher

Reply:
@sriyen-msft I saw some other publishers which are companies like 
![image](https://user-images.githubusercontent.com/114684189/195535374-78bffdd4-47b8-452c-9eae-ebb2ed6ba95a.png)
or 
![image](https://user-images.githubusercontent.com/114684189/195535549-41914589-617f-43b0-bcc0-bc7e1e956be4.png)

Reply:
@marpedroche thank you so much for the clarification. You are correct is this case the publisher is Axazure. 
May I ask you please to resolve all other issues. 
Thank you so much.